src/gluonnlp/model/transformer.py

```diff
@@ -106,7 +107,8 @@ def __init__(self, units=512, hidden_size=2048, dropout=0.0, use_residual=True,
                                   weight_initializer=weight_initializer,
                                   bias_initializer=bias_initializer,
                                   prefix='ffn_2_')
-            self.dropout_layer = nn.Dropout(dropout)
```

Comment:
Might as well change 
https://github.com/dmlc/gluon-nlp/blob/972d8666b287e0e58a6273adf186307b79f2e2ba/src/gluonnlp/model/transformer.py#L207
https://github.com/dmlc/gluon-nlp/blob/972d8666b287e0e58a6273adf186307b79f2e2ba/src/gluonnlp/model/transformer.py#L339
https://github.com/dmlc/gluon-nlp/blob/972d8666b287e0e58a6273adf186307b79f2e2ba/src/gluonnlp/model/transformer.py#L719
and 
https://github.com/dmlc/gluon-nlp/blob/972d8666b287e0e58a6273adf186307b79f2e2ba/src/gluonnlp/model/transformer.py#L854
?

Reply:
Sure. Will change them accordingly.

Reply:
Thanks. Would you mind also adding the case when dropout is set to 0 in the unit test? https://github.com/dmlc/gluon-nlp/blob/master/tests/unittest/test_models.py#L75